extensions/aerospace/src/switchToWorkspace.tsx

```diff
@@ -0,0 +1,59 @@
+import { Action, ActionPanel, List, showToast, Toast } from "@raycast/api";
+import { getWorkspaces, switchToWorkspace } from "./utils/appSwitcher";
+import { useEffect, useState } from "react";
+
+export default function Command() {
+  const [workspaces, setWorkspaces] = useState<string[]>([]);
```

Comment:
we might want to use `useCachedState` here, please check [switchApps.tsx](https://github.com/raycast/extensions/blob/main/extensions/aerospace/src/switchApps.tsx#L13)